services/web/server/src/simcore_service_webserver/meta_modeling_handlers.py

```diff
@@ -75,27 +75,29 @@ async def _get_project_iterations_range(
                     tag.name, return_none_if_fails=True
                 ):
                     if iteration:
-                        raise NotTaggedAsIteration(
+                        raise NotTaggedAsIterationException(
```

Comment:
another though: why call it Exception and not Error? or when would you do that? anyway I'm ok with this

Reply:
The name was a suggewtion from @GitHK :-)